Gemfile

```diff
@@ -52,6 +52,10 @@ gem 'devise-token_authenticatable'
 gem 'jwt', '~> 2.2'
 gem 'oauth2', '~> 1.4.7' # Used for Stripe Connect
 
+gem 'omniauth'
+gem 'omniauth_openid_connect', github: 'andrewpbrett/omniauth_openid_connect', branch: "omniauth2"
```

Comment:
This gem specifies a dependency on omniauth 1.9, which has a security issue. Updating the dependency to 2.0 seems to work and the repository has an open PR to do so. For the time being I forked the repo and pointed the Gemfile at my own fork.